packages/terra-switch/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "terra-switch",
+  "main": "lib/Switch.js",
+  "version": "1.0.0",
```

Comment:
This should be `0.0.0` until it's released

Reply:
[updated](https://github.com/cerner/terra-core/pull/2846/commits/f4692ff31e65e7a746c1e364be845ab3210915a6#diff-dc7c6a7043881aec37938f976b58bb7f)